php-practice_1.php

```diff
@@ -1,43 +1,143 @@
 <?php
 
+// Q1
+
+$name='畠山';
+echo '私の名前は「'.$name.'」です';
```

Comment:
[nits]
`=`や`.`の前後は必ず空白を設けるようにしましょう！
```suggestion
$name = '畠山';
echo '私の名前は「' . $name . '」です';
```

Reply:
空白を開けました。